test/scripts/measures/2018/fixtures/test-quality-measures.csv

```diff
@@ -66,11 +66,11 @@ Formula to determine the overall performance rate, given multiple strata of perf
+Possible values: simpleAverage, weightedAverage, sumNumerators, overallStratumOnly",measureSpecification,,,,,
+,,,,,,,,,,,,methods,,,,,,measureSets,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+title,eMeasureId,nqfEMeasureId,nqfId,measureId,description,nationalQualityStrategyDomain,measureType,primarySteward,methodsClaims,methodsCertifiedSurveyVendor,methodsElectronicHealthRecord,methodsCmsWebInterface,methodsAdministrativeClaims,methodsRegistry,allergyImmunology,anesthesiology,cardiology,electrophysiologyCardiacSpecialist,gastroenterology,dermatology,emergencyMedicine,generalPracticeFamilyMedicine,internalMedicine,obstetricsGynecology,ophthalmology,orthopedicSurgery,otolaryngology,pathology,pediatrics,physicalMedicine,plasticSurgery,preventiveMedicine,neurology,mentalBehavioralHealth,diagnosticRadiology,interventionalRadiology,vascularSurgery,generalSurgery,thoracicSurgery,urology,generalOncology,radiationOncology,hospitalists,rheumatology,nephrology,infectiousDisease,neurosurgical,podiatry,dentistry,metricType,firstPerformanceYear,lastPerformanceYear,eMeasureUuid,isHighPriority,isInverse,numStrataClaims,numStrataRegistry,numStrataEcqm,overallAlgorithm,default,registry,claims,cmsWebInterface,isIcdImpacted,isToppedOutByProgram
+Diabetes: Hemoglobin A1c (HbA1c) Poor Control (>9%),CMS122v6,N/A,59,1,Percentage of patients 18-75 years of age with diabetes who had hemoglobin A1c > 9.0% during the measurement period,Effective Clinical Care,Intermediate Outcome,National Committee for Quality Assurance,X,-,X,X,-,X,-,-,-,-,-,-,-,X,X,-,-,-,-,-,-,-,-,X,-,-,-,-,-,-,-,-,-,-,-,-,X,-,-,-,-,singlePerformanceRate,2017,NULL,,TRUE,TRUE,1,1,1,,,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_001_Registry.pdf,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_001_Claims.pdf,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Web-Interface-Measures/2018_WI_DM.pdf,FALSE,FALSE
+Coronary Artery Disease (CAD): Beta-Blocker Therapy - Prior Myocardial Infarction (MI) or Left Ventricular Systolic Dysfunction (LVEF < 40%),CMS145v6,N/A,70,7,Percentage of patients aged 18 years and older with a diagnosis of coronary artery disease seen within a 12-month period who also have a prior MI OR a current or prior LVEF <40% who were prescribed beta-blocker therapy,Effective Clinical Care,Process,Physician Consortium for Performance Improvement,-,-,X,-,-,X,-,-,X,-,-,-,-,X,X,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,multiPerformanceRate,2017,NULL,,FALSE,FALSE,NULL,2,2,weightedAverage,,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_007_Registry.pdf,,,TRUE,FALSE
```

Comment:
Github is not being helpful here... but what changed in these lines? I don't see any differences...?

Reply:
encoding difference I think, seems to be fixed now